tools/license-builder.sh

```diff
@@ -29,7 +29,7 @@ fi
 
 
 # Dependencies bundled in distributions
-addlicense "Acorn" "deps/acorn" "$(cat ${rootdir}/deps/acorn/LICENSE)"
+addlicense "Acorn" "deps/acorn" "$(cat ${rootdir}/deps/acorn/acorn/LICENSE)"
```

Comment:
Do we need to add `acorn-walk` separately?

Reply:
I'm not sure.
acorn is a monorepo: https://github.com/acornjs/acorn
The license files are exactly the same, and I changed the license-builder to say `deps/acorn`, which includes both packages.